backend/test/request_handler_test.py

```diff
@@ -26,56 +30,90 @@ def setUp(cls):
             [(RequestHandlerTest.REQUEST_URI, RequestHandler),
              ], debug=True)
         cls.testapp = cls.webtest.TestApp(app)
-        initModels(cls)
 
-    @patch('utils.verify_token', return_value={'email': 'useradmin@test.com'})
+    @patch('utils.verify_token', return_value=ADMIN)
     def test_get(self, verify_token):
         """Test method get of RequestHandler."""
-        response = self.testapp.get('/api/requests/' + self.request.key.urlsafe() + '/user')
-        request = json.loads(response._app_iter[0])
+        admin = mocks.create_user(ADMIN['email'])
```

Comment:
We need to think how to solve the recurrent repetition of code even after mocks creation. I think we could create some scenarios like an institution with admin and hierarchy.